float64 x
float64 y
float64 intensity
float64 radius
int32 identity
float64 certainty
